src/addandremove.js

```diff
@@ -0,0 +1,113 @@
+const taskContainer = document.querySelector('.list-Container');
+
+class CreatetodoList {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
+  }
+
+  static displayTasks = (task, oneTwo, three) => {
+    const listItem = document.createElement('li');
+    listItem.id = task.id;
+    listItem.className = 'lists';
+    listItem.innerHTML = `
+          <div class="list-Container">
+          <input type="checkbox" name="" id="${task.id}" class="checkbox" ${oneTwo}>
+          <p class="${three}" id="${task.id}">${task.description}</p>
+          </div>
+          
+          <div class="trash">
+          
+          <i id="pen" class="fa-solid fa-pen"></i>
+          <i id="delete" class="fa-solid fa-trash-can"></i>
+          </div>
+          `;
+
+    taskContainer.appendChild(listItem);
+  };
+
+  static loadFromLocalStorage() {
+    let tasks;
+
+    if (localStorage.getItem('TasksInfo')) {
+      tasks = JSON.parse(localStorage.getItem('TasksInfo'));
+    } else {
+      tasks = [];
+    }
+    return tasks;
+  }
+
+  static displayTasksOnPage() {
+    const tasks = CreatetodoList.loadFromLocalStorage();
+    tasks.forEach((task) => {
+      if (task.completed === true) {
+        CreatetodoList.displayTasks(task, 'checked', 'tickedItem');
+      } else {
+        CreatetodoList.displayTasks(task, '/', 'none');
+      }
+    });
+  }
```

Comment:
- [OPTIONAL]Based on the requirements, you ought to create a list of todos in an array and render those todos on the webpage. For this milestone, the tasks ought to be static. I would advise you to implement this so that you can fully implement all features for this milestone. 👍 🖱️ 

![image](https://github.com/siddghosh108/Todo-List/assets/77106962/05221098-44b3-4db6-ab89-d71ce4b71307)